File: TrendAnalyzerServer/src/main/java/vorotilenko/trendanalyzerserver/server/ServerMessageTypes.java
package vorotilenko.trendanalyzerserver.server;

/**
 * Types of JSON messages which are sent from the {@link WSServerEndpoint} to the client
 */
public class ServerMessageTypes {
    /**
     * Message with initial data
     */
    public static final int INIT = 0;
    /**
     * Normal message
     */
    public static final int NORMAL_MESSAGE = 1;
    /**
     * String with information about client-server interaction
     */
    public static final int INFO = 2;
}

File: TrendAnalyzer/app/src/main/java/vorotilenko/trendanalyzer/serverinteraction/ServerMessageTypes.kt
package vorotilenko.trendanalyzer.serverinteraction

/**
 * Types of JSON messages which are sent from the server to the [WSClientEndpoint].
 */
object ServerMessageTypes {
    /**
     * Message with initial data.
     */
    const val INIT = 0
    /**
     * Normal message.
     */
    const val NORMAL_MESSAGE = 1
    /**
     * String with information about client-server interaction.
     */
    const val INFO = 2
}